packages/jaeger-ui/tsconfig.lint.json

```diff
@@ -30,7 +30,6 @@
     "src/api/jaeger.js",
     "src/components/SearchTracePage/SearchResults/ResultItem.markers.js",
     "src/model/order-by.js",
-    "src/model/trace-viewer.js",
```

Comment:
I will be linted since this config file [inherits another config](https://github.com/jaegertracing/jaeger-ui/blob/e968cdb07c61dedaaf3e1dfda888d8ef596b0491/packages/jaeger-ui/tsconfig.lint.json#L23) with [include](https://github.com/jaegertracing/jaeger-ui/blob/e968cdb07c61dedaaf3e1dfda888d8ef596b0491/packages/jaeger-ui/tsconfig.json#L23) option.
Please correct me if I'm wrong.